pkgs/development/lua-modules/overrides.nix

```
@@ -544,6 +544,17 @@ in
       '';
   });
 
+  lze  = prev.lze.overrideAttrs(oa: {
+    doCheck = lua.luaversion == "5.1";
+    nativeCheckInputs = [ final.nlua final.busted ];
+    checkPhase = ''
+      runHook preCheck
+      export HOME=$(mktemp -d)
+      busted --lua=nlua
+      runHook postCheck
+      '';
```

```suggestion
    '';
```
Here you go, this is my contribution to this work X)